src/app/components/home/home.component.html

```diff
@@ -1,5 +1,4 @@
 <div class="row">
-  <app-search-user></app-search-user>
-  <app-show-user></app-show-user>
+  <app-user-search></app-user-search>
```

Comment:
After the name change here will be something like `<app-user-container></app-user-conatiner>`